apps/microsoft-teams/app-actions/src/constants.ts

```diff
@@ -5,4 +5,5 @@ export const TOPIC_ACTION_MAP = {
   'ContentManagement.Entry.publish': 'published',
   'ContentManagement.Entry.unpublish': 'unpublished',
   'ContentManagement.Entry.delete': 'deleted',
```

Comment:
This update is a little confusing to me... I don't think we necessarily have a concept of a workflow that has a value... if that makes sense.  There aren't named changes in state in workflows, it's just a transition from one step to the next. 

For that reason, I just copied the Topic key also to its value.  I'd definitely be interested to figure out any ways to improve this code.

cc: @jsdalton